frontend/src/components/WofFooter.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <footer>
+        <img class="footer__ul-img" src="../assets/logo-ul-eng.png">
+        <div class="footer__authors">
+            <h4>Authors:</h4>
+            <span>Karol Kozłowski</span>
+            <span>Konrad Pagacz</span>
+            <span>Robert Szczurkowski</span>
+            <span>Magdalena Wiechecka</span>
+            <span>Kacper Fryczak</span>
```

Comment:
Do we plan on having anything else in the footer? It's quite high for the amount of content it currently has:
![image](https://user-images.githubusercontent.com/24480246/140993374-a249fbc2-7fc2-4113-b907-1e38f1559690.png)

Is there a point in maybe spreading names horizontally or does it go against good ux practice?

Reply:
I left some space for possible content